tests/ops/qutrit/test_qutrit_non_parametric_ops.py

```diff
@@ -168,7 +173,7 @@ def test_no_pow_ops(self, op):
         assert len(op_pow) == 1
         assert op_pow[0].__class__ == op.__class__
 
-        pows = [0.1, 2, -2, -2.5]
+        pows = [0.1, -2.5]
```

Comment:
```suggestion
        pows = [0.1, -2, -2.5]
```

Reply:
`-2` should not raise `PowUndefinedError` since `-2 % 4 = 2` is what gets passed into `pow` which is now defined. Maybe this test is not necessary since `THadamard` is tested in `test_period_two_three_noninteger_power` :thinking:

Reply:
Where is the `%4`?

Reply:
Oh I just realized that this is a test for qutrit ops